lua/tardis/screens/sh_virtualconsole.lua

```diff
@@ -235,5 +272,150 @@ TARDIS:AddScreen("Controls", {menu=false}, function(self,ext,int,frame,screen)
 			end
 		end
 	end
+end
 
+local function new_virtual_console(self,ext,int,frame,screen)
+	frame:SetBackgroundColor(Color(0,0,0))
+
+	--background
+
+	local background=vgui.Create("DImage", frame)
+	local theme = TARDIS.visualgui_theme_basefolder
+	theme = theme..TARDIS:GetSetting("visual_gui_theme").."/"
+	background:SetImage(theme.."background.png")
+	background:SetSize( frame:GetWide(), frame:GetTall() )
+
+	local layout_rows
+	if screen.is3D2D
+	then
+		layout_rows = math.floor(TARDIS:GetSetting("visual_gui_screen_numrows"))
+	else
+		layout_rows = math.floor(TARDIS:GetSetting("visual_gui_popup_numrows"))
+	end
+	local layout = HexagonalLayout:new(frame, layout_rows, 0.15)
+
+	-- controls
+
+	local door = TardisScreenButton:new(frame)
+	door:SetIsToggle(true)
+	door:SetText("door")
```

Comment:
These controls should have better labels for textonly theme

Reply:
textonly is mainly made for debugging, not to be used
I am planning to replace it with images with text and no icons

Reply:
Can we hide this textonly option for now then? Maybe by removing the json file for the moment or having a enabled: false in the JSON

Reply:
Textonly theme has been completely removed for now in 35a660969efd2005259750628580676698b2b45d
Json has been completely removed too, since it's incompatible with workshop

Issue moved to:
https://github.com/MattJeanes/TARDIS/issues/360#issuecomment-784149887